pkg/cluster/spec/grafana.go

```diff
@@ -160,13 +164,22 @@ func (i *GrafanaInstance) InitConfig(
 		return err
 	}
 
+	topo := reflect.ValueOf(i.topo)
+	if topo.Kind() == reflect.Ptr {
+		topo = topo.Elem()
+	}
+	val := topo.FieldByName("Monitors")
+	if (val == reflect.Value{}) {
```

Comment:
Is this check necessary? There is check for monitor list length just below.

Reply:
This sentinel checks if the struct contains field `Monitors` or not, if not, then `val.Interface()` will panic